Makefile

```diff
@@ -27,8 +27,8 @@ test: $(COMPILED_CAIRO0_PROGRAMS)
 test_metal: $(COMPILED_CAIRO0_PROGRAMS)
 	cargo test -F metal
 
-coverage: $(COMPILED_CAIRO0_PROGRAMS)
-	cargo llvm-cov --lcov --output-path lcov.info
+coverage: $(COMPILED_PROGRAMS)
```

Comment:
This should be `COMPILED_CAIRO0_PROGRAMS`

Reply:
Done